sandbox/rocky/tf/plotter/plotter.py

```diff
@@ -0,0 +1,74 @@
+import atexit
+from queue import Empty,Queue
+from rllab.sampler.utils import rollout
+import numpy as np
+from threading import Thread
+import tensorflow as tf
+
+__all__ = [
+    'init_worker',
+    'init_plot',
+    'update_plot',
+    'shutdown_worker'
+]
+
+process = None
+queue = None
+
+class PlotterThread(Thread):
+    def __init__(self, queue,sess):
+        super(PlotterThread,self).__init__()
+        self.queue=queue
+        self.sess = sess
+
+    def run(self):
+        env = None
+        policy = None
+        max_length = None
+        while True:
+            msgs = {}
+            # Only fetch the last message of each type
+            with self.sess.as_default():    
+                with self.sess.graph.as_default():
+                    while True:                    
+                        try:
+                            msg = self.queue.get_nowait()
+                            msgs[msg[0]] = msg[1:]
+                        except Empty:
+                            break
+                    if 'stop' in msgs:
+                        break
+                    elif 'update' in msgs:
+                        env, policy = msgs['update']
+                        # env.start_viewer()
```

Comment:
Please do not commit commented-out code.

Reply:
Done